frontend/javascripts/oxalis/view/left-border-tabs/layer_settings_tab.tsx

```diff
@@ -457,6 +473,9 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
           <div className="flex-item">
             {hasHistogram && !isDisabled ? this.getEditMinMaxButton(layerName, isInEditMode) : null}
           </div>
+          <div className="flex-item">
+            {isVolumeTracing && !isDisabled ? this.getMergeWithFallbackLayerButton(layer) : null}
```

Comment:
Not every volume annotation has a fallback layer. Shouldn't this be checked before showing this button?